examples/logger.py

```diff
@@ -133,6 +136,29 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
         logger.addHandler(self.handler)
+        self.initLevelSelector(self.loggerFrame.levelSelector)
+
+    def initLevelSelector(self, box: QComboBox):
+        """Initializes a combo box for selecting level."""
+        box.addItem("DEBUG")
```

Comment:
Referred to [addItems()](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QComboBox.html#PySide2.QtWidgets.PySide2.QtWidgets.QComboBox.addItems), you can add all items in just a line!